src/category.rs

```diff
@@ -388,29 +388,17 @@ pub fn show(req: &mut Request) -> CargoResult<Response> {
 
 /// Handles the `GET /category_slugs` route.
 pub fn slugs(req: &mut Request) -> CargoResult<Response> {
-    let conn = req.tx()?;
-    let stmt = conn.prepare(
-        "SELECT slug FROM categories \
-         ORDER BY slug",
-    )?;
-    let rows = stmt.query(&[])?;
+    let conn = req.db_conn()?;
+    let slugs = categories::table
+        .select((categories::slug, categories::slug))
```

Comment:
I'd still like the order by to ensure that the slugs are alphabetical, please :)